13-students-doing-data-science.Rmd

```diff
@@ -1 +1,146 @@
+- [Teaching Tech Together](http://teachtogether.tech/#the-rules): a book on teaching programming
+- [Data Science for Undergraduates](https://www.nap.edu/catalog/25104/data-science-for-undergraduates-opportunities-and-options): a report on undergraduate data science education
+
+## General strategies related to teaching and learning
+
+The National Academy of Science commissioned a report, [*How People Learn*](https://www.nap.edu/catalog/9853/how-people-learn-brain-mind-experience-and-school-expanded-edition), that aimed to summarize research from the educational psychology and the learning sciences on teaching and learning. In 2018, the book was updated in [*How People Learn II*](https://www.nap.edu/catalog/24783/how-people-learn-ii-learners-contexts-and-cultures), which aimed to emphasize the social and cultural aspects of teaching and learning, which were not as much the focus of the earlier report. In this section, we highlight general strategies related to teaching and learning from the latest report (abbreviated as *HPL2*), with an emphasis on strategies applicable to teaching and learning data science. 
+
+### Teaching and learning are complex
+
+One principle that *HPL2* begins with is that learning is complex. in short, learning is not just about what learners know, or think, but is also about developmental, cultural, contextual, and historical factors - and distinctions between individuals with respect to each of these factors. This is an asset to teachers, as the authors of the report state: learners bring resources that can serve as a starting point for their learning trajectory when it comes to data science. These distinctions also mean that educators need to be concerned with - and to consider within their purview - factors well beyond what learners know, but also what their prior educational experiences have been and what resources and other individuals they have access to at work and at home, for example.
+
+<!-- Not satisifed with this. -->
+
+### Learners learn many different things (consciously and unconsciously)
+
+We often think of learning in terms of objectives for specific lessons, but learners learn many different things at different times. The authors of *HPL2* point out that individuals learn in response to different challenges and circumstances, including those in formal learning environements, such as workshops or classes. This principle implies a strategy that involves supporting learners to do data science, however and whenever they learn it. This means that it is both okay - and even to be expected - that learners may take away more from a problem they try to solve on their own, than what they do from a workshop or class (or even a degree!). This also suggests that learners may learn things that we do not anticipate, such as how intructors try to solve problems that arise in class.
+
+### Metacognition is important (even though it sounds more sophisticated than it is!)
```

Comment:
I'm wondering how we can model metacognition throughout the text using callouts, etc.

Reply:
added to notes